pkg/kubernetes/client.js

```diff
@@ -505,7 +528,7 @@ define([
                 debug("kube config scheme:", scheme);
                 schemes.unshift(scheme);
             })
-            .always(step);
+            .always(step("/oapi"));
```

Comment:
This runst the step function here once while registering the undefined return value from step("/oapi") as a callback. This seems like a bug.